hack/make-rules/test-cmd-util.sh

```diff
@@ -1460,7 +1460,7 @@ run_kubectl_get_tests() {
   kube::test::if_has_string "${output_message}" 'valid-pod' # pod details
   output_message=$(kubectl get pods/valid-pod -o name -w --request-timeout=1 "${kube_flags[@]}")
   kube::test::if_has_not_string "${output_message}" 'STATUS' # no headers
-  kube::test::if_has_string     "${output_message}" 'pods/valid-pod' # resource name
+  kube::test::if_has_string     "${output_message}" 'Pod./valid-pod' # resource name
```

Comment:
Output still shows caps and a trailing dot.  You need to output the toLower'd strings and eliminate the trailing dots.

Reply:
Thanks, will update tests